tests/data_set/test_data_set_constants.py

```diff
@@ -40,23 +40,29 @@ def get_sample(sample_file: str) -> Union[str, bytes]:
 )
 
 # Extract Data Set
-TEST_EXTRACT_DATA_SET_RESULT_BASE_SUCCESS_XML = get_sample(
-    "extract_data_set_result_base_success.xml"
+TEST_EXTRACT_DATA_SET_RESULT_BASE_ONLY_SUCCESS_XML = get_sample(
+    "extract_data_set_result_base_only_success.xml"
 )
-TEST_EXTRACT_DATA_SET_RESULT_BASE_SUCCESS_DICTIONARY = get_sample(
-    "extract_data_set_result_base_success.json"
+TEST_EXTRACT_DATA_SET_RESULT_BASE_ONLY_SUCCESS_DICTIONARY = get_sample(
+    "extract_data_set_result_base_only_success.json"
 )
-TEST_EXTRACT_DATA_SET_RESULT_GENERIC_BASE_SUCCESS_XML = get_sample(
-    "extract_data_set_result_generic_base_success.xml"
+TEST_EXTRACT_DATA_SET_RESULT_GENERIC_BASE_ONLY_SUCCESS_XML = get_sample(
+    "extract_data_set_result_generic_base_only_success.xml"
 )
-TEST_EXTRACT_DATA_SET_RESULT_GENERIC_BASE_SUCCESS_DICTIONARY = get_sample(
-    "extract_data_set_result_generic_base_success.json"
+TEST_EXTRACT_DATA_SET_RESULT_GENERIC_BASE_ONLY_SUCCESS_DICTIONARY = get_sample(
+    "extract_data_set_result_generic_base_only_success.json"
 )
-TEST_EXTRACT_DATA_SET_RESULT_BASE_ERROR_XML = get_sample(
-    "extract_data_set_result_base_error.xml"
+TEST_EXTRACT_DATA_SET_RESULT_BASE_ONLY_ERROR_XML = get_sample(
+    "extract_data_set_result_base_only_error.xml"
 )
-TEST_EXTRACT_DATA_SET_RESULT_BASE_ERROR_DICTIONARY = get_sample(
-    "extract_data_set_result_base_error.json"
+TEST_EXTRACT_DATA_SET_RESULT_BASE_ONLY_ERROR_DICTIONARY = get_sample(
+    "extract_data_set_result_base_only_error.json"
+)
+TEST_EXTRACT_DATA_SET_RESULT_bad_ATTRIBUTE_ERROR_XML = get_sample(
+    "extract_data_set_result_bad_attribute_error.xml"
+)
+TEST_EXTRACT_DATA_SET_RESULT_bad_ATTRIBUTE_ERROR_DICTIONARY = get_sample(
```

Comment:
Constants should not contain lowercase letters.

Reply:
caused by an imperfect find/replace. Now resolved.